ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -1107,5 +1107,19 @@
     "EndButtonLabel": "Save and continue",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-documentsv2/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-documentsv2/submitted"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "manageGuardianReports",
+    "Name": "Manage Guardian Report",
+    "Description": "Manage Guardian Report",
+    "DisplayOrder": 1000,
+    "PreConditionState(s)": "Open;Submitted;Gatekeeping;GATEKEEPING_LISTING;PREPARE_FOR_HEARING;FINAL_HEARING;CLOSED;Deleted;RETURNED",
```

Comment:
Guess we don't need to enable it for `open` state. At the moment, Cafcass will not receive case in `open` state